stdlib/3/tempfile.pyi

```diff
@@ -5,7 +5,9 @@
 
 import sys
 from types import TracebackType
-from typing import Any, AnyStr, Generic, IO, Iterable, Iterator, List, Optional, overload, Tuple, Type
+from typing import Any, AnyStr, Generic, IO, Iterable, Iterator, List, Optional, overload, Tuple, Type, Union
+
+from os import path as path
```

Comment:
This would re-export `path` from `tempfile`. Is there any reason you didn't use `from os.path import _PathType`?